spec/lib/tasks/properties_spec.rb

```diff
@@ -1,80 +1,73 @@
-require 'rake'
-require 'stringio'
+require "rake"
+require "stringio"
```

Comment:
Sorry for the extra noise. I've noticed there's a mixed-use of double quotes and single quotes. I tend to use double quotes so I updated the rest of the file to match. Happy to chat about this in case anyone feels strongly about which style we use.